stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -622,7 +622,6 @@ function(_compile_swift_files
   endif()
 
   list(APPEND swift_flags "-enable-experimental-feature" "NoncopyableGenerics2")
-  list(APPEND swift_flags "-enable-experimental-feature" "SuppressedAssociatedTypes")
```

Comment:
I think it would be clearer to leave this in, personally. Also, usually we solve this class of problems by relaxing the diagnostics if the source file kind is a swiftinterface, rather than by enabling the feature.